CODEOWNERS

```diff
@@ -7,3 +7,4 @@
 # Unless a later match takes precedence, they will be requested for review
 # when someone opens a Pull Request.
 *   @redhat-partner-solutions/red-hat-crucible
```

Comment:
Should this be [crucible-owners](https://github.com/orgs/redhat-partner-solutions/teams/crucible-owners) ?

Reply:
Looks good now ` @redhat-partner-solutions/crucible-owners`.